cmd/policy-obligations.go

```diff
@@ -332,6 +333,73 @@ func policyDeleteObligationValue(cmd *cobra.Command, args []string) {
 	HandleSuccess(cmd, id, t, val)
 }
 
+// ****
+// Obligation Triggers
+// ****
+func policyCreateObligationTrigger(cmd *cobra.Command, args []string) {
+	c := cli.New(cmd, args)
+	h := NewHandler(c)
+	defer h.Close()
+
+	ctx := cmd.Context()
+	attributeValue := c.Flags.GetRequiredString("attribute-value")
+	action := c.Flags.GetRequiredString("action")
+	obligationValue := c.Flags.GetRequiredString("obligation-value")
+	clientID := c.Flags.GetOptionalString("client-id")
+	metadataLabels = c.Flags.GetStringSlice("label", metadataLabels, cli.FlagsStringSliceOptions{Min: 0})
+
+	trigger, err := h.CreateObligationTrigger(ctx, attributeValue, action, obligationValue, clientID, getMetadataMutable(metadataLabels))
+	if err != nil {
+		cli.ExitWithError("Failed to create obligation trigger", err)
+	}
+
+	rows := getObligationTriggerRows(trigger)
+	t := cli.NewTabular(rows...)
+	HandleSuccess(cmd, trigger.GetId(), t, trigger)
+}
+
+func policyDeleteObligationTrigger(cmd *cobra.Command, args []string) {
+	c := cli.New(cmd, args)
+	h := NewHandler(c)
+	defer h.Close()
+
+	id := c.Flags.GetRequiredString("id")
+	force := c.Flags.GetOptionalBool("force")
+	ctx := cmd.Context()
+
+	cli.ConfirmAction(cli.ActionDelete, "obligation trigger", id, force)
+
+	resp, err := h.DeleteObligationTrigger(ctx, id)
+	if err != nil {
+		errMsg := fmt.Sprintf("Failed to delete obligation trigger (%s)", id)
+		cli.ExitWithError(errMsg, err)
+	}
+
+	rows := [][]string{
+		{"Id", id},
+	}
+	t := cli.NewTabular(rows...)
+	HandleSuccess(cmd, id, t, resp)
+}
+
+func buildObligationValueFQN(namespaceFQN, obligationName, value string) string {
+	return fmt.Sprintf("%s/obl/%s/value/%s", namespaceFQN, obligationName, value)
+}
+
+func getObligationTriggerRows(trigger *policy.ObligationTrigger) [][]string {
+	rows := [][]string{
+		{"Id", trigger.GetId()},
+		{"Attribute Value FQN", trigger.GetAttributeValue().GetFqn()},
+		{"Action FQN", trigger.GetAction().GetName()},
```

Comment:
I'd avoid `fqn` for action as it's just `name` in the protos with no qualifier like `https://` in front name-scheme wise.